scripts/themes/index.js

```diff
@@ -21,16 +21,21 @@ const { value: themeMap } = renderSync({
 const themes = Object.keys(themeMap);
 
 // Generates the data for themes and tokens to template with Handlebars.
-const tokens = Object.keys(themeMap[themes[0]].value).map(token => ({
-  themes: [],
-  token,
-}));
+const tokens = Object.keys(themeMap[themes[0]].value)
+  // TODO - currently only handling color tokens.
```

Comment:
I don't initially understand what this comment and the comment on line 37 is describing - Is there another task to be completed here?

Reply:
From this comment (and possibly related Slack comments earlier this month): https://github.com/carbon-design-system/ibm-security/issues/288#issuecomment-570322154

> It may just be best to ignore any non-color-related variable for now and only show color variables in themes.md

That is, we may want to include these non-color tokens from the theme map in the future. But for now, the easiest thing is to filter them out.

Reply:
Thanks for clarifying - I recommend removing `TODO` from the comment here to avoid confusion, and being more explicit about the code. For example - 

```suggestion
  // Filter only color tokens from the theme.
```

Reply:
Wait so you want to remove ‘TODO’ which we use elsewhere to help us find things that need to be revisited, and only describe what the line does in the comment? 😅 (Also consider above you asked for a helper ‘filterColor’ so that seems redundant?